synapse/models/crypto.py

```diff
@@ -34,6 +34,20 @@ def getModelDefs(self):
                 ('crypto:smart:contract', ('guid', {}), {
                     'doc': 'A smart contract.',
                 }),
+                ('crypto:smart:effect:transfertoken', ('guid', {}), {
+                    'doc': 'A smart contract effect which transfers ownership of a non-fungible token.',
+                }),
+                ('crypto:smart:effect:transfertokens', ('guid', {}), {
+                    'doc': 'A smart contract effect which transfers fungible tokens.',
+                }),
+                ('crypto:smart:effect:supplytokens', ('guid', {}), {
```

Comment:
Supply makes me think tokens are being added or increased only. Thoughts on creating two forms to represent this concept - crypto:smart:effect:supplyincr and crypto:smart:effect:supplydecr for supply increase and decrease?